SDWebImage/Core/SDImageCoderHelper.m

```diff
@@ -257,22 +258,26 @@ + (CGImageRef)CGImageCreateDecoded:(CGImageRef)cgImage orientation:(CGImagePrope
     }
     
     BOOL hasAlpha = [self CGImageContainsAlpha:cgImage];
-    // iOS prefer BGRA8888 (premultiplied) or BGRX8888 bitmapInfo for screen rendering, which is same as `UIGraphicsBeginImageContext()` or `- [CALayer drawInContext:]`
-    // Though you can use any supported bitmapInfo (see: https://developer.apple.com/library/content/documentation/GraphicsImaging/Conceptual/drawingwithquartz2d/dq_context/dq_context.html#//apple_ref/doc/uid/TP30001066-CH203-BCIBHHBB ) and let Core Graphics reorder it when you call `CGContextDrawImage`
-    // But since our build-in coders use this bitmapInfo, this can have a little performance benefit
-    CGBitmapInfo bitmapInfo = kCGBitmapByteOrder32Host;
-    bitmapInfo |= hasAlpha ? kCGImageAlphaPremultipliedFirst : kCGImageAlphaNoneSkipFirst;
-    CGContextRef context = CGBitmapContextCreate(NULL, newWidth, newHeight, 8, 0, [self colorSpaceGetDeviceRGB], bitmapInfo);
-    if (!context) {
-        return NULL;
-    }
     
-    // Apply transform
-    CGAffineTransform transform = SDCGContextTransformFromOrientation(orientation, CGSizeMake(newWidth, newHeight));
-    CGContextConcatCTM(context, transform);
-    CGContextDrawImage(context, CGRectMake(0, 0, width, height), cgImage); // The rect is bounding box of CGImage, don't swap width & height
-    CGImageRef newImageRef = CGBitmapContextCreateImage(context);
-    CGContextRelease(context);
+    // Using UIGraphicsRenderer on supported platforms, and use the main screeen's perferred format (like Wide Color)
```

Comment:
TODO: Check whether it's suitable to change (Performance benchmark)

Reply:
@kinarobin The benchmark shows that using `UIGraphicsImageRenderer` for this background decode case, does not help for performance.

The reason is that the Core Animation always perferred BGRA/BGRX (4 channels), not anything like 3 or 1 channels. Which means, the RAM usage when `UIGraphicsImageRenderer` or low level `CGContext` code is the same.

And, using `UIGraphicsImageRenderer` will cause generate dummy autorelease object of `UIImage` (because it's UIKit high level API). Which does not cause much performance cost, but have a compared drop down to the low level CGContext code.

So, I just revert this changes.